lora/README.md

```diff
@@ -0,0 +1,3 @@
+# LoRA
```

Comment:
Would be good to have the command to generate an engine with the appropriate flags for LoRA

Reply:
I think we can link https://github.com/triton-inference-server/tensorrtllm_backend/blob/b93b948be37046cf2a8a222067771fb366baf2a8/inflight_batcher_llm/README.md#running-lora-inference-with-inflight-batching